Sources/Markdown/Interpretive Nodes/Aside.swift

```diff
@@ -21,87 +21,155 @@ import Foundation
         /// An "experiment" aside.
-        case experiment = "Experiment"
-
+        public static let experiment = Kind(displayName: "Experiment")
+        
         /// A "warning" aside.
-        case warning = "Warning"
-
+        public static let warning = Kind(displayName: "Warning")
+        
         /// An "attention" aside.
-        case attention = "Attention"
-
+        public static let attention = Kind(displayName: "Attention")
+        
         /// An "author" aside.
-        case author = "Author"
-
+        public static let author = Kind(displayName: "Author")
+        
         /// An "authors" aside.
-        case authors = "Authors"
-
+        public static let authors = Kind(displayName: "Authors")
+        
         /// A "bug" aside.
-        case bug = "Bug"
-
+        public static let bug = Kind(displayName: "Bug")
+        
         /// A "complexity" aside.
-        case complexity = "Complexity"
-
+        public static let complexity = Kind(displayName: "Complexity")
+        
         /// A "copyright" aside.
-        case copyright = "Copyright"
-
+        public static let copyright = Kind(displayName: "Copyright")
+        
         /// A "date" aside.
-        case date = "Date"
-
+        public static let date = Kind(displayName: "Date")
+        
         /// An "invariant" aside.
-        case invariant = "Invariant"
-
+        public static let invariant = Kind(displayName: "Invariant")
+        
         /// A "mutatingVariant" aside.
-        case mutatingVariant = "MutatingVariant"
-
+        public static let mutatingVariant = Kind(displayName: "Mutating Variant")
+        
         /// A "nonMutatingVariant" aside.
-        case nonMutatingVariant = "NonMutatingVariant"
-
+        public static let nonMutatingVariant = Kind(displayName: "Non-Mutating Variant")
+        
         /// A "postcondition" aside.
-        case postcondition = "Postcondition"
-
+        public static let postcondition = Kind(displayName: "Postcondition")
+        
         /// A "precondition" aside.
-        case precondition = "Precondition"
-
+        public static let precondition = Kind(displayName: "Precondition")
+        
         /// A "remark" aside.
-        case remark = "Remark"
-
+        public static let remark = Kind(displayName: "Remark")
+        
         /// A "requires" aside.
-        case requires = "Requires"
-
+        public static let requires = Kind(displayName: "Requires")
+        
         /// A "since" aside.
-        case since = "Since"
-
+        public static let since = Kind(displayName: "Since")
+        
         /// A "todo" aside.
-        case todo = "ToDo"
-
+        public static let todo = Kind(displayName: "To Do")
+        
         /// A "version" aside.
-        case version = "Version"
-
+        public static let version = Kind(displayName: "Version")
+        
         /// A "throws" aside.
-        case `throws` = "Throws"
-
-        public init?(rawValue: String) {
```

Comment:
Actually, I think we should keep this initializer and mark it as deprecated so that clients have time to adopt `init(displayName:)` instead.

Reply:
Great suggestion @Kyle-Ye. I've pushed changes that take this approach instead.